src/Symfony/Component/Validator/Resources/translations/validators.ur.xlf

```diff
@@ -0,0 +1,407 @@
+            </trans-unit>
+            <trans-unit id="84">
+                <source>This value should be a multiple of {{ compared_value }}.</source>
+                <target>کا ضرب ہوني چاہیے۔ {{ compared_value }} یہ ويليو </target>
+            </trans-unit>
+            <trans-unit id="85">
+                <source>This Business Identifier Code (BIC) is not associated with IBAN {{ iban }}.</source>
+                <target> سے وابستہ نہیں ہے۔ IBAN {{ iban }} (BIC) یہ کاروباری شناختی کوڈ</target>
+            </trans-unit>
+            <trans-unit id="86">
+                <source>This value should be valid JSON.</source>
+                <target> ہونی چاہیے JSON یہ ويليو درست</target>
+            </trans-unit>
+            <trans-unit id="87">
+                <source>This collection should contain only unique elements.</source>
+                <target>یہ مجموعہ صرف منفرد عناصر پر مشتمل ہونا چاہیے۔</target>
+            </trans-unit>
+            <trans-unit id="88">
+                <source>This value should be positive.</source>
+                <target>یہ ويليو مثبت ہونی چاہیے۔</target>
+            </trans-unit>
+            <trans-unit id="89">
+                <source>This value should be either positive or zero.</source>
+                <target>یہ ويليو یا تو مثبت یا صفر ہونی چاہیے۔</target>
+            </trans-unit>
+            <trans-unit id="90">
+                <source>This value should be negative.</source>
+                <target>یہ ويليو منفی ہونی چاہیے۔</target>
+            </trans-unit>
+            <trans-unit id="91">
+                <source>This value should be either negative or zero.</source>
+                <target>یہ ويليو یا تو منفی یا صفر ہونی چاہیے</target>
+            </trans-unit>
+            <trans-unit id="92">
+                <source>This value is not a valid timezone.</source>
+                <target>یہ ويليو درست ٹائم زون نہیں ہے۔</target>
+            </trans-unit>
+            <trans-unit id="93">
+                <source>This password has been leaked in a data breach, it must not be used. Please use another password.</source>
+                <target>یہ پاس ورڈ ڈیٹا کی خلاف ورزی میں لیک ہو گیا ہے، اسے استعمال نہیں کرنا چاہیے۔ براۓ کرم دوسرا پاس ورڈ استعمال کریں۔</target>
+            </trans-unit>
+            <trans-unit id="94">
+                <source>This value should be between {{ min }} and {{ max }}.</source>
+                <target> کے درمیان ہونی چاہیے۔ {{ max }} اور {{ min }} یہ ويليو</target>
+            </trans-unit>
+            <trans-unit id="95">
+                <source>This value is not a valid hostname.</source>
+                <target> نہیں ہے۔ hostname یہ ويليو درست </target>
+            </trans-unit>
+            <trans-unit id="96">
+                <source>The number of elements in this collection should be a multiple of {{ compared_value }}.</source>
+                <target> کی ضرب ہونی چاہیے۔ {{ compared_value }} اس مجموعہ میں عناصر کی تعداد </target>
```

Comment:
```suggestion
                <target>کی ضرب ہونی چاہیے۔ {{ compared_value }} اس مجموعہ میں عناصر کی تعداد</target>
```

It looks like some translations are whitespaces that should be removed. Here is one such example. Can you remove them all?

Reply:
Sure sir. 
No problem I can fix them and submit an other PR.

Reply:
@MuhammadAakash if possible, please try to make those changes in this PR so we can merge everything at once. Thank you!

Reply:
@javiereguiluz 
Sure.

Reply:
Hi, sir @fabpot @javiereguiluz. Hopefully, you are doing great. I have removed all the extra spaces from the files.
Please review.
Thanks.